Mage.Sets/src/mage/cards/m/MwonvuliOoze.java

```diff
@@ -68,7 +69,7 @@ public MwonvuliOozePTValue copy() {
     }
 
     @Override
-    public String getMessage() {
+    public String getMessage(ValuePhrasing phrasing) {
         return "1 plus twice the number of age counters on it";
```

Comment:
interesting, I'm curious if this would work as IntPlusDynamicValue(1, MultipliedValue(CountersSourceCount))

Reply:
100% it would, i'm planning to convert this and a few more custom values that either:

- are the same as two or more other custom values such that it should be commonized
- use modifiers like this (even if they dont own a base value or multiplier)